vagrant/multi_node/hosts/localmesos-cluster

```diff
@@ -1,25 +1,25 @@
 #inventory file for local multi-node cluster environment
 
 [machines]
-localmesos[01:03] env=test base_data_dir=/data shared_mount=/vagrant
+localmesos[02:04] env=test base_data_dir=/data shared_mount=/vagrant
 
```

Comment:
Good call. This way /etc/hosts doesn't have to change when going between single and mutli